### ISSUE TITLE
Examples of types of interactions we value/want to promote

### ISSUE THREAD

**Issue by j-ro** [opensupporter/osdi-cc#1]
In line with your emailed comment about this not only being a list of donts, figure some kind of list of examples of types of interactions we value or want to explicitly promote might be good. I don't have specifics yet, but figured I'd open an issue as a discussion point...

**Comment by sonyabea** [comment]
Yes! That's sort of what I have in mind for the "Oops" and "Ouch" sections. I'd love to have some language people can look at to use when they're ouched or they've oopsed.
